apiv2/urls.py

```diff
@@ -97,7 +99,8 @@
 
     # Oauth2
     url(r'^oauth2/', include('apiv2.oauth2_urls', namespace='oauth2_provider')),
-    url(r'^login/$', LoginView.as_view(template_name='api/minimal_login.html'), name="api-login"),
+    url(r'^login/$', login, {'template_name': 'api/minimal_login.html',
+                             'authentication_form': FsAuthenticationForm}, name="api-login"),
```

Comment:
Yes, this means that logins through API have always been case-sensitive. The difference is case-sensitiveness and (from now on) email option.